src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольник
+
+    public Rectangle(String color,double firsSide, double secondSide) {
+        super(color, firsSide, secondSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return getFirsSide() * getSecondSide();
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Длина = " + getFirsSide()
+                + " Ширина = " + getSecondSide()
+                + " Площадь прямоугольника = " + detAreaOfFigure());
+    }
+
+    @Override
+    public String infoFromRigure() {
```

Comment:
The method name `infoFromRigure` seems to be a typo. It should likely be `infoFromFigure` to match the naming convention and avoid confusion.